scripts.d/50-vvenc.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+SCRIPT_REPO="https://github.com/fraunhoferhhi/vvenc"
+
+ffbuild_enabled() {
+    [[ $ADDINS_STR == *4.4* ]] && return -1
+    [[ $ADDINS_STR == *5.0* ]] && return -1
+    [[ $ADDINS_STR == *5.1* ]] && return -1
+    [[ $ADDINS_STR == *6.0* ]] && return -1
+    [[ $ADDINS_STR == *7.0* ]] && return -1
+    return 0
+}
+
+fixarm64=()
```

Comment:
please use a local variable in the function

Reply:
I replaced with export flag when target is arm64.

Reply:
You just had to move this into the function and declare it local.
The current approach with the export seems much more ugly and it also does not need to be exported.

Reply:
I moved and declare the local function. Replaced export by local flag.
If you're still need to add codes, it would be helpful by making patch file using diff. :)

- Martin Eesmaa